create history subject page (#4345)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/explore-subjects/history.md

---
title: Become a history teacher
subject: history
title_paragraph: |-

>   <p>As a history teacher, you'll bring the past to life by introducing pupils to key events and significant people. By encouraging the use of critical thinking, you’ll help pupils form their own opinions and uncover new perspectives.</p>

Reviewer: ```suggestion
  <p>As a history teacher, you'll bring the past to life by introducing pupils to key events and significant people from history. By encouraging critical thinking, you’ll help pupils form their own opinions and uncover new perspectives.</p>
```

Reviewer: Just a suggestion, maybe 'significant people from the past', just think it needs to say what kind of significant people

---
title: Become a history teacher
subject: history
title_paragraph: |-
  <p>As a history teacher, you'll bring the past to life by introducing pupils to key events and characters that have shaped our present.</p>
  <p>

>   By encouraging the use of critical thinking, you’ll help pupils form their own opinions and uncover new perspectives on the past</p>

Reviewer 2: The word "past" appears twice in this section - is it worth tweaking slightly?

>   By encouraging the use of critical thinking, you’ll help pupils form their own opinions and uncover new perspectives on the past</p>

Reviewer: Looks like it's still missing a full stop?
